frontend/src/components/sitemanager/SiteManagerHomepage.js

```diff
@@ -113,7 +240,14 @@ class SiteManagerHomepage extends Component {
                             <PopupMealTotals weekArr= {weekArr} day={3} showModal={this.handleOpenModal}/>
                             <PopupMealTotals weekArr= {weekArr} day={4} showModal={this.handleOpenModal}/>
                         </div>
-                        <button className="route" style={{marginTop: 20,width: 300}} onClick={this.printDocument}>Print Meal Totals</button>
+                        <div className = "confirmation-buttons" style={{ display:'flex', marginTop: 20}}>
+                            <h3>Driver Routes For: </h3>
+                            <button className="route" onClick={() => this.printDocument("SLO", "M", 0)}>Monday</button>
+                            <button className="route" onClick={() => this.printDocument("SLO", "T", 1)}>Tuesday</button>
+                            <button className="route" onClick={() => this.printDocument("SLO", "W", 2)}>Wednesday</button>
+                            <button className="route" onClick={() => this.printDocument("SLO", "Th", 3)}>Thursday</button>
+                            <button className="route" onClick={() => this.printDocument("SLO", "F", 4)}>Friday</button>
```

Comment:
You're hardcoding "SLO" as the site, replace with localStorage.get("site")

Reply:
Updated. Thanks, wasn't sure where to get that information